sway-core/src/semantic_analysis/namespace/root.rs

```diff
@@ -859,8 +859,12 @@ impl Root {
                         current_mod_path.push(ident.clone());
                     }
                     None => {
-                        decl_opt =
-                            Some(self.resolve_symbol_helper(handler, engines, ident, module)?);
+                        decl_opt = Some(
+                            module
+                                .root_lexical_scope()
```

Comment:
Shouldn't this be `current_lexical_scope()`?

Or is the idea that if we're resolving a symbol from `Root`, then we're asking for a top-level symbol in the module? If so, shouldn't this method be moved to `Module` and renamed `resolve_root_symbol_and_mod_path`?